python/typemap_utils.cpp

```diff
@@ -53,6 +53,16 @@ static PyObject *py_meep_src_time_object() {
     return src_time;
 }
 
+static PyObject *py_material_object() {
+    static PyObject *material_object = NULL;
+    if(material_object == NULL) {
```

Comment:
most of the rest of the code puts a space between `if` and the paren

Reply:
Fixed (in entire file).